java-client/src/main/java/co/elastic/clients/util/Pair.java

```diff
@@ -56,7 +56,8 @@ public static <K, V> Pair<K, V> of(K name, V value) {
         return new Pair<>(name, value);
     }
 
-    public static <K, V> JsonpDeserializer<Pair<K, V>> deserializer(Function<String, K> keyDeserializer, JsonpDeserializer<V> valueDeserializer) {
+    public static <K, V> JsonpDeserializer<Pair<K, V>> deserializer(Function<String, K> keyDeserializer,
```

Comment:
Please avoid reformatting unrelated files in the PR.

Reply:
Looks like the checkstyle finding corrected here was fixed in the following [commit](https://github.com/elastic/elasticsearch-java/commit/323498f8214c03fc0eb8edff9ab3542e2d5e07bf#diff-45e77db90a1ba56a54a38cec1b93c14e49b069b1f11f7cb843adad3c399f38d1R59).  Rebase corrected the issue.